src/components/Interpretations/InterpretationModal/InterpretationThread.js

```diff
@@ -33,7 +33,9 @@ const InterpretationThread = ({
                     <IconClock16 color={colors.grey700} />
                     {moment(interpretation.created).format('LLL')}
                 </div>
-                <DownloadMenu relativePeriodDate={interpretation.created} />
+                {DownloadMenu && (
```

Comment:
We should add some space below the interpretation date.

Reply:
Fixed:
<img width="335" alt="Screenshot 2022-08-23 at 11 04 48" src="https://user-images.githubusercontent.com/150978/186118472-e11ffc8b-d5e6-4de4-a3a7-52f190408675.png">